src/modules/user/docUtils/user.responsedoc.ts

```diff
@@ -123,3 +127,29 @@ export class EnrolledCourseResponseBody {
    */
   courseId: string;
 }
+
+export class getAllGamifiedResponseBody {
+  /**
+   * First name of the user
+   * @example 'John'
+   */
+  first_name: string;
+
+  /**
+   * Last name of the user
+   * @example 'Doe'
+   */
+  last_name: string;
+
+  /**
+   * Score of the user
+   * @example 100
+   */
+  score: number;
+
+  /**
+   * The photo url
+   * @example 'https://g.gle/mypic.jpeg'
```

Comment:
Add a link to dummy image that works, people copy doc fields without thinking anything so the links mentioned should work.
https://cdn.pixabay.com/photo/2015/03/04/22/35/head-659652_1280.png